Lib/gftools/builder/dependencies.py

```diff
@@ -0,0 +1,163 @@
+"""
+Similar to pip freeze, but only listing the installed dependencies for the selected package.
+
+E.g.:
+    $ python freeze_requirements.py fonttools[ufo]
+    # Installed requirements for 'fonttools[ufo]' (5 in total):
+
+    appdirs==1.4.4
+    fonttools==4.24.4
+    fs==2.4.13
+    pytz==2021.1
+    six==1.16.0
+
+Credit to Cosimo Lupo (anthrotype) for starting this as a gist:
+https://gist.github.com/anthrotype/531a425c8a0ba5ee975bc2ec8add7b82
+"""
+
+__requires__ = ["packaging", "importlib_metadata; python_version < '3.8'"]
```

Comment:
you don't need this line, I added back then because I was fond of this [pip-run](https://github.com/jaraco/pip-run) tool but here it serves no purpose since gftools deps are declared in its setup.py